frontend/src/components/Forms/AddMinespaceUser.js

```diff
@@ -38,6 +45,8 @@ export const AddMinespaceUser = (props) => (
               placeholder="Please Select a Mine"
               component={renderConfig.MULTI_SELECT}
               data={props.mines}
+              onChange={props.handleChange}
```

Comment:
This one seems unnecessary, since events on the input handle both the default and the filtered search.

Reply:
Multi-select keeps everything focused, so onChange is needed.